app/models/concerns/legal_aid_application_state_machine.rb

```diff
@@ -17,7 +17,7 @@ module LegalAidApplicationStateMachine
       state :provider_checked_citizens_means_answers
       state :checking_merits_answers
       state :checked_merits_answers
-      state :merits_completed
+      state :assessment_submitted
```

Comment:
Do we need a run an SQL query like this?
`update legal_aid_applications set state = 'assessment_submitted' where state = 'merits_completed'`

It might not be necessary if we are allowed to simply delete all applications in staging/prod

Reply:
`LegalAidApplication.where(state: 'merits_completed').update_all(state: 'assessment_submitted')`

Good point!